CHANGELOG.md

```diff
@@ -2,11 +2,17 @@
 
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
 
+## 0.15.0 - 2020-09-14
+### Added
+- Attach PostgreSQL servers by connection string
+- PostgreSQL support for integration with App Service extension
+### Fixed
+- [Bugs fixed] (https://github.com/microsoft/vscode-cosmosdb/issues?q=is%3Aopen+is%3Aissue+milestone%3A0.15.0)
```

Comment:
You should model this after previous links. It's supposed to reference closed issues, not open ones